cmd/syncthing/connections.go

```diff
@@ -374,3 +325,19 @@ func (s *connectionSvc) CommitConfiguration(from, to config.Configuration) bool
 
 	return true
 }
+
+func setTCPOptions(conn *net.TCPConn) {
+	var err error
+	if err = conn.SetLinger(0); err != nil {
+		l.Infoln(err)
+	}
+	if err = conn.SetNoDelay(false); err != nil {
+		l.Infoln(err)
+	}
+	if err = conn.SetKeepAlivePeriod(60 * time.Second); err != nil {
```

Comment:
(re: IRC discussion yesterday) Huh, apparently we do at least try to set these and I don't recall any bugs in the Infoln()s that should have been printed if these failed... Would be interesting to some day look at a tcpdump of an idle connections and see if this actually does what it says on the tin on all relevant platforms... Then we could skip the ping/pong stuff potentially.